mk/rabbitmq-dist.mk

```diff
@@ -2,10 +2,22 @@
 
 DIST_DIR = plugins
 
+# MIX = "echo y | mix"
+
 dist_verbose_0 = @echo " DIST  " $@;
 dist_verbose_2 = set -x;
 dist_verbose = $(dist_verbose_$(V))
 
+ifeq ($(MIX_ARCHIVES),)
+MIX_ARCHIVES = $(wildcard ~/.mix/archives)
+endif
+
+mix_task_archive_deps_installed = $(lastword $(wildcard $(MIX_ARCHIVES)/mix_task_archive_deps-*))
```

Comment:
Ok, I will test.

Just to be more specific, I had issues with `$(wildcard)` when dealing with an absolute path in the home directory. There is no issue with other `$(wildcard)` limited to a relative subdirectory of the project because we only use US-ASCII there.